lib/assets/connection_cell/main.js

```diff
@@ -237,49 +335,17 @@ export function init(ctx, info) {
         inputClass="input"
         :grow
       />
-      <div class="input-icon-container grow">
-        <BaseInput
-          v-if="fields.use_password_secret"
-          name="password_secret"
-          label="Password"
-          v-model="fields.password_secret"
-          inputClass="input input-icon"
-          :grow
-          readonly
-          @click="selectSecret"
-        />
-        <BaseInput
-          v-else
-          name="password"
-          label="Password"
-          type="text"
-          v-model="fields.password"
-          inputClass="input input-icon-text"
-          :grow
-        />
-        <div class="icon-container">
-          <label class="hidden-checkbox">
-            <input
-              type="checkbox"
-              :value="fields.use_password_secret"
-              name="use_password_secret"
-              v-model="fields.use_password_secret"
-              class="hidden-checkbox-input"
-            />
-            <svg v-if="fields.use_password_secret" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"
-                 width="22" height="22">
-              <path fill="none" d="M0 0h24v24H0z"/>
-              <path d="M18 8h2a1 1 0 0 1 1 1v12a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1V9a1 1 0 0 1 1-1h2V7a6 6 0 1 1 12 0v1zM5
-                10v10h14V10H5zm6 4h2v2h-2v-2zm-4 0h2v2H7v-2zm8 0h2v2h-2v-2zm1-6V7a4 4 0 1 0-8 0v1h8z" fill="#000"/>
-            </svg>
-            <svg v-else xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24">
-              <path fill="none" d="M0 0h24v24H0z"/>
-              <path d="M21 3v18H3V3h18zm-8.001 3h-2L6.6 17h2.154l1.199-3h4.09l1.201 3h2.155l-4.4-11zm-1 2.885L13.244
-                12h-2.492l1.247-3.115z" fill="#445668"/>
-            </svg>
-          </label>
-        </div>
-      </div>
+      <BaseSecret
+        inputName="password"
+        secretName="password_secret"
+        toggleName="use_password_secret"
```

Comment:
`secretName` and `secretValue` can be confusing because we already have secret with name and value (in fact `secretValue` is the secret name :laughing:). Perhaps `secretInputName`, `textInputName`, `secretInputValue`, `textInputValue`, or something similarly explicit?